src/components/Database/DatabasePicker.scss

```diff
@@ -24,11 +24,11 @@ a.mdc-list-item.mdc-list-item--activated {
 
 .Database-Picker {
   background-color: #fafafa;
-  border-radius: 16px 0 0 16px;
   border-right: solid 1px #dcdcdc;
 
   .mdc-list-group__subheader {
-    margin: 0 16px;
+    // Align horizontally with list icon and vertically with CardActionBar.
+    margin: 8px 16px 11px;
```

Comment:
Generally we want multiples of 4px only, and at least one axis (in this case vertical) should usually use flex. Can this be flexed with only minimal alignment for icons?

Having said that, also happy for you to leave as is, and I will do a pass on alignment soon.

Reply:
This one cannot be flexed since it's just a paragraph, but let's definite fix the alignment later once we have the correct font size.